torch/csrc/jit/fuser/tensor_desc.h

```diff
@@ -41,8 +41,11 @@ struct TORCH_API TensorDesc {
   TensorDesc(const at::Tensor& t)
       : TensorDesc(t.scalar_type(), t.sizes(), t.strides()) {}
 
-  TensorDesc(const c10::CompleteTensorTypePtr& type)
-      : TensorDesc(type->scalarType(), type->sizes(), type->strides()) {}
+  TensorDesc(const c10::ProfiledTensorTypePtr& type)
+      : TensorDesc(
```

Comment:
nitpick: probably won't hurt to stick in `INTERNAL_TORCH_ASSERT(type->isComplete())` in places like this